diesel/src/sqlite/expression/expression_methods.rs

```diff
@@ -10,8 +10,10 @@ use crate::sql_types::SqlType;
 pub trait SqliteExpressionMethods: Expression + Sized {
     /// Creates a Sqlite `IS` expression.
     ///
-    /// This behaves identically to the `=` operator, except that `NULL` is
-    /// treated as a normal value.
+    /// The `IS` operator work like = except when one or both of the operands are NULL.
+    /// In this case, if both operands are NULL, then the `IS` operator evaluates to 1 (true).
+    /// If one operand is NULL and the other is not, then the `IS` operator evaluates to 0 (false).
```

Comment:
As diesel handles the `0`-> `false`, `1` -> `true` mapping here I would prefer to just use the boolean values here:
```suggestion
    /// In this case, if both operands are NULL, then the `IS` operator evaluates to true.
    /// If one operand is NULL and the other is not, then the `IS` operator evaluates to false.
```